### PR TITLE
chore(plugin-compat): add extensions for react-bootstrap-table2-paginator and react-draggable

### PR DESCRIPTION
**Pull request by naser-baig-au4**
**What's the problem this PR addresses?**

Adding new entries to `plugin-compat database` with the PRs I sent upstream.

### CODE REVIEW

File: packages/plugin-compat/sources/extensions.ts [hunk 1]
```diff
@@ -149,4 +149,17 @@ export const packageExtensions: Array<[string, any]> = [
...
+    dependencies: {
+      "classnames": "^2.2.6"
+    },
+  }],
+  // https://github.com/STRML/react-draggable/pull/525
+  [`react-draggable@*`, {
+    dependencies: {
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1"
+    },
```
Comment (merceyz) [inline packages/plugin-compat/sources/extensions.ts:163]: These should most likely be peer dependencies

**Review by merceyz — CHANGES_REQUESTED** [review]

File: packages/plugin-compat/sources/extensions.ts (hunk 1 above)
Comment (naser-baig-au4) [reply packages/plugin-compat/sources/extensions.ts:163]: Hi @merceyz,

The error I got in PnP while I was working with that package mentioned that the package was looking for react and react-dom as dependencies. But if you think that adding them in peerDependencies makes sense, then I will make that change 👍

Comment (merceyz) [reply packages/plugin-compat/sources/extensions.ts]: Yeah, `react` and `react-dom` are usually peer dependencies but PnP doesn't know that, it just knows it's undeclared

Comment (naser-baig-au4) [reply packages/plugin-compat/sources/extensions.ts]: Understood. Changes made upstream :)

**Review by merceyz — APPROVED** [review]

**Comment by merceyz** [comment]
Thanks!

**Pull request merged**